full/src/main/java/apoc/uuid/Uuid.java

```diff
@@ -111,7 +103,18 @@ public static class UuidInstallInfo extends UuidInfo {
 
     }
 
-    private String getUuidFunctionName() {
+    public static Map<String, Object> setExistingNodes(GraphDatabaseService db, Pools pools, String label, UuidConfig uuidConfig) {
+        final String uuidFunctionName = getUuidFunctionName();
+
+        return Util.inTx(db, pools, txInThread ->
+                txInThread.execute("CALL apoc.periodic.iterate(" +
+                                "\"MATCH (n:" + Util.sanitizeAndQuote(label) + ") RETURN n\",\n" +
```

Comment:
change this to return id(n) 
also do `WHERE n.uuidProperty IS NULL` 
and lookup the id in the update statement
otherwise it will leak transactions